src/pages/workspace/WorkspaceInvitePage.js

```diff
@@ -161,172 +190,142 @@ class WorkspaceInvitePage extends React.Component {
+    const policyName = lodashGet(props.policy, 'name');
+
+    return (
+        <ScreenWrapper shouldEnableMaxHeight>
+            <FullPageNotFoundView
+                shouldShow={_.isEmpty(props.policy)}
+                onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS_WORKSPACES)}
+            >
+                <HeaderWithBackButton
+                    title={props.translate('workspace.invite.invitePeople')}
+                    subtitle={policyName}
+                    shouldShowGetAssistanceButton
+                    guidesCallTaskID={CONST.GUIDES_CALL_TASK_IDS.WORKSPACE_MEMBERS}
+                    onBackButtonPress={() => {
+                        clearErrors();
+                        Navigation.goBack(ROUTES.getWorkspaceMembersRoute(props.route.params.policyID));
+                    }}
+                />
+                <View style={[styles.flexGrow1, styles.flexShrink0, styles.flexBasisAuto]}>
+                    <OptionsSelector
+                        contentContainerStyles={[styles.flexGrow1, styles.flexShrink0, styles.flexBasisAuto]}
+                        listContainerStyles={[styles.flexGrow1, styles.flexShrink1, styles.flexBasis0]}
+                        canSelectMultipleOptions
+                        sections={sections}
+                        selectedOptions={selectedOptions}
+                        value={searchTerm}
+                        shouldShowOptions={OptionsListUtils.isPersonalDetailsReady(props.personalDetails)}
+                        onSelectRow={toggleOption}
+                        onChangeText={updateOptionsWithSearchTerm}
+                        onConfirmSelection={inviteUser}
+                        headerMessage={headerMessage}
+                        hideSectionHeaders
+                        boldStyle
+                        shouldFocusOnSelectRow={!Browser.isMobile()}
+                        textInputLabel={props.translate('optionsSelector.nameEmailOrPhoneNumber')}
+                    />
+                </View>
+                <View style={[styles.flexShrink0]}>
+                    <FormAlertWithSubmitButton
+                        isDisabled={!selectedOptions.length}
+                        isAlertVisible={getShouldShowAlertPrompt()}
+                        buttonText={props.translate('common.next')}
+                        onSubmit={inviteUser}
+                        message={props.policy.alertMessage}
+                        containerStyles={[styles.flexReset, styles.flexGrow0, styles.flexShrink0, styles.flexBasisAuto, styles.mb5]}
+                        enabledWhenOffline
+                        disablePressOnEnter
+                    />
+                </View>
+            </FullPageNotFoundView>
+        </ScreenWrapper>
+    );
 }
 
 WorkspaceInvitePage.propTypes = propTypes;
 WorkspaceInvitePage.defaultProps = defaultProps;
+WorkspaceInvitePage.defaultName = 'WorkspaceInvitePage';
 
 export default compose(
     withLocalize,
```

Comment:
Please use the `useLocalize` hook instead of `withLocalize` HOC in functional components.

Reply:
Also, can you please clear this one a bit more if possible.

Reply:
> Please use the `useLocalize` hook instead of `withLocalize` HOC in functional components.

In a bit more detail, please if possible.